.processes/development.md

```diff
@@ -153,9 +153,22 @@ Newly created issues ([labelled as](#labels): `new issue`) may not all have been
 
 In `Retrospective` we aim to summarize the results of the last spring, in order to help us identify and fix issues in our processes and company culture.
 
-- what went better than expected
-- what went worse than expected
-- lessons learned
+The process is using [EasyRetro](https://easyretro.io/). `Retrospective` follows this schedule:
+
+1. Everybody secretly adds hidden items to `What Went Well` column.
+2. The moderator reveals the `What Went Well` column.
+3. Everybody has ~30 secs to speak about their items.
```

Comment:
I get that we want to keep the discussions focused but I think 30s is too short. But unlimited bullets isn't helpful either. So I'd limit to max 3 bullets per person in each of the two categories and 30s speaking time per bullet.